src/preamble.js

```diff
@@ -37,7 +37,7 @@ if (typeof WebAssembly !== 'object') {
 #if ASSERTIONS
   abort('No WebAssembly support found. Build with -s WASM=0 to target JavaScript instead.');
 #else
-  err('no native wasm support detected');
+  abort('no native wasm support detected');
```

Comment:
Looks like we should merge these two aborts (by dropping the former abort and the if else).

Reply:
Ok. I have updated the code.